metadata/ and logs/ dirs viewable in SIP arrange

jrwdunham opened the pull request:
UI has "Show metadata & logs directories?" checkbox to control whether
these directories will be displayed when browsing transfers in backlog.

sevein commented on src/dashboard/src/media/js/ingest/backlog.js:
```diff
@@ -61,6 +61,9 @@ function renderBacklogSearchForm(search_uri, on_success, on_error) {
       // Query Django, which queries ElasticSearch, to get the backlog file info
       var query_url = search_uri + '?' + search.toUrlParams();
+      if (!$('#id_show_metadata_logs').is(':checked')) {
+          query_url = query_url + '&hidemetadatalogs=1';
```
Should we use [`jQuery.param`](http://api.jquery.com/jquery.param/) to make sure that we're building the query string consistently? Source: https://goo.gl/RfVJkl. It may not be necessary.

jrwdunham replied:
Ok.

sevein commented on src/dashboard/src/components/ingest/views.py:
```diff
@@ -504,9 +504,16 @@ def transfer_backlog(request, ui):
+    # (for now).
+    BACKLOG_FILTER = elasticSearchFunctions.BACKLOG_FILTER
+    if ui == 'appraisal' or request.GET.get('hidemetadatalogs'):
+        BACKLOG_FILTER = elasticSearchFunctions.BACKLOG_FILTER_NO_MD_LOGS
```
If `BACKLOG_FILTER` is not a constant perhaps we shouldn't use capitals. What do you think?

jrwdunham replied:
Makes sense.

jrwdunham committed:
metadata/ and logs/ dirs viewable in SIP arrange

UI has "Show metadata & logs directories?" checkbox to control whether
these directories will be displayed when browsing transfers in backlog.

sevein commented:
Cool! I think we think of safer ways to build URLs (e.g. we've used URI.js in other projects) - but I'm not sure if it's really needed. Thanks for the changes! 👍

jhsimpson commented:
:+1:  from me too.

jrwdunham commented:
Cherry-picked to qa/1.x as d22d8a711316ec9978db70dda9d36f6fc95dfb81
